### ISSUE TITLE
A4/Portal fail test

### ISSUE THREAD

**Issue by gabryelreyes** [cyberbotics/webots#6599]
**Describe the Bug**
`tests/test_worlds.py` reports unwanted warnings for file `projects/robots/a4/portal/worlds/portal_pe.wbt`:
```log
WARNING: Portal "Portal" > Pose  > DEF GEAR_JOINT HingeJoint > RotationalMotor "portal_motor::gear": too big requested position: 1.2 > 12
```

**Steps to Reproduce**
1. Open world file `projects/robots/a4/portal/worlds/portal_pe.wbt`

**Expected behavior**
No warnings

**Comment by gabryelreyes** [comment]
Error is present in R2023b too.
Setting the position to 0 does not appear to have an effect on the world.

**Comment by gabryelreyes** [comment]
Fixed with 989419224d49f7f28f2a0061a71c9368d1b9f270
